variables.tf

```diff
@@ -300,6 +300,12 @@ variable "cluster_addons" {
   default     = {}
 }
 
+variable "vpc_cni_addon" {
+  description = "vpc-cni addon definition"
+  type        = any
+  default     = {}
```

Comment:
Could this object be typed instead of any ?

Reply:
If this variable is typed, its attributes need to be optional in order to set an empty default value, otherwise we'd have to set a defaukt verison and configuration_values

Reply:
You can have a field typed and nullable ?